### ISSUE TITLE
Add text stylization options

### ISSUE THREAD

**Issue by zachspears** [cadin/panels#6]
It would be really neat/helpful to be able to include some basic text stylization such as sizing, **bolding**, and _italics_. Thanks!

**Comment by cadin** [comment]
Can you say more about what you're trying to do?

You should be able to set \*bold\* and \_italic\_ using asterisks and underscores in your strings as detailed in the [Playdate SDK docs](https://sdk.play.date/1.12.2/Inside%20Playdate.html#_drawing_text). Or you can set the layer's font to a bold or italic variant if you want the entire string to have that formatting.

You can set a custom font in Panels for [individual text layers](https://cadin.github.io/panels/docs/comic-data/layers/#font) or [for all layers in a panel](https://cadin.github.io/panels/docs/comic-data/panels.html#font). That's how you would change size since pixel fonts are only designed for a single size.

It would look something like this:
```
{ text = "Hello world!", font="fonts/font-Cuberick-Bold" }
```

I may need to add the ability to set an entire [font family](https://sdk.play.date/1.12.2/Inside%20Playdate.html#f-graphics.setFontFamily) to enable multiple formats (bold and italic) within a layer with a custom font. I'll look into that.
But currently you can for sure do a single custom font per layer, or multiple formats within a layer using the default font.

**Comment by cadin** [comment]
@zachspears 
I added support for a `fontFamily` property. You can add this to a single text layer, or specify the family to use for the entire panel or sequence, just like when specifying a [`font`](https://cadin.github.io/panels/docs/comic-data/layers/#font).

You can specify a family very similar to how you would [create a font family using the SDK](https://sdk.play.date/1.12.3/Inside%20Playdate.html#f-graphics.font.newFamily).

Send the `fontFamily` property a table with the appropriate path to mapped to the constant for each variant.

Example text layer:
```lua
{ text = "Normal *Bold*, _Italic_", x = 16, y = 180,
    fontFamily = {
        [Panels.Font.NORMAL] = "fonts/SasserSlab/Sasser-Slab",
        [Panels.Font.BOLD] = "fonts/SasserSlab/Sasser-Slab-Bold",
        [Panels.Font.ITALIC] = "fonts/SasserSlab/Sasser-Slab-Italic"
    },
},
```

This change will be included in the next release (and added to the docs). For now you can grab it from the [develop branch](https://github.com/cadin/panels/tree/develop).

**Comment by cadin** [comment]
This change is now merged into main and released as [v1.1](https://github.com/cadin/panels/releases/tag/v1.1)
